test/functional/maxblocksinflight.py

```diff
@@ -1,93 +0,0 @@
-#!/usr/bin/env python3
-# Copyright (c) 2015-2016 The Bitcoin Core developers
-# Distributed under the MIT software license, see the accompanying
-# file COPYING or http://www.opensource.org/licenses/mit-license.php.
-"""Test nodes responses to having many blocks in flight.
-
-In this test we connect to one node over p2p, send it numerous inv's, and
-compare the resulting number of getdata requests to a max allowed value.  We
-test for exceeding 128 blocks in flight, which was the limit an 0.9 client will
-reach. [0.10 clients shouldn't request more than 16 from a single peer.]
-"""
-
-from test_framework.mininode import *
-from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import *
-
-MAX_REQUESTS = 128
-
-class TestManager(NodeConnCB):
-    def on_getdata(self, conn, message):
-        self.log.debug("got getdata %s" % repr(message))
-        # Log the requests
-        for inv in message.inv:
-            if inv.hash not in self.blockReqCounts:
-                self.blockReqCounts[inv.hash] = 0
-            self.blockReqCounts[inv.hash] += 1
```

Comment:
If I'm understanding correctly, this method doesn't get called anymore since `getdata` isn't the response to `inv`?